cunumeric/linalg/cholesky.py

```diff
@@ -202,11 +202,46 @@ def tril(context: Context, p_output: StorePartition, n: int) -> None:
     task.execute()
 
 
+def _batched_cholesky(output: DeferredArray, input: DeferredArray) -> None:
+    # the only feasible implementation for right now is that
+    # each cholesky submatrix fits on a single proc. We will have
+    # wildly varying memory available depending on the system.
+    # Just use a fixed cutoff to provide some sensible warning.
+    # TODO: find a better way to inform the user dims are too big
+    context = output.context
+    task = context.create_auto_task(CuNumericOpCode.BATCHED_CHOLESKY)
+    task.add_input(input.base)
+    task.add_output(output.base)
+    ndim = input.base.ndim
+    task.add_broadcast(input.base, (ndim - 2, ndim - 1))
+    task.add_broadcast(output.base, (ndim - 2, ndim - 1))
```

Comment:
I think you need one of this broadcast or alignment below, both is redundant

Reply:
Probably this is redundant, but I don't think it's incorrect? Personal preference is to be more explicit about the pattern

Reply:
I take it back, my suggestion is actually incorrect. Without the alignment, the partitioner could technically partition the input and output differently along the first ndim - 2 dimensions.